src/casus/interpriter/BooleanListValue.js

```diff
@@ -0,0 +1,40 @@
+//@flow strict
+
+import IntValue from './IntValue.js';
+import BooleanValue from './BooleanValue.js';
+import {verifyBoolean, defaultValueFor} from './Value.js';
+
+class BooleanListValue {
+	list: Array<BooleanValue>;
+
+	constructor() {
+		this.list=[];
+	}
+
+	setAt(index: IntValue, to:BooleanValue): void {
+		if (index.val<0 || index.val>this.list.length) {
+			//just ignore it
```

Comment:
Is there more to be added here? If not then inverse the comparative operators and remove the else.